src/staging/staging_main.py

```diff
@@ -143,14 +143,23 @@ def run_staging(
 
         # Write both snapshots to feather file at given path
         if is_network:
-            helpers.write_snapshot_to_feather(
+            feather_fname = f"{snapshot_name}_corrected.feather"
+            s_feather_fname = f"{secondary_snapshot_name}.feather"
+            helpers.df_to_feather(
                 feather_path,
-                snapshot_name,
+                feather_fname,
                 full_responses,
-                write_feather,
-                secondary_snapshot_name,
+                write_feather
+
+            )
+            helpers.df_to_feather(
```

Comment:
If secondary_full_responses = None, the pipeline fails when it tries to write to feather